src/utils/search.js

```diff
@@ -1,10 +1,9 @@
 // This helper goes and fetches Good First Issues
 // This module expects a search query which is just a standard GitHub search query. These queries should not include sort or order, since those are defined in the function.
 const octokit = require("@octokit/rest")({
-  timeout: 0, // 0 means no request timeout
-  headers: {
-    accept: "application/vnd.github.v3+json",
-    "user-agent": "octokit/rest.js v1.2.3" // v1.2.3 will be current version
+  userAgent: "octokit/rest.js v1.2.3",
```

Comment:
Hi i am new to open source can you tell me from where you figured out headers needs to be removed and user-agent needs to be replaced by userAgent?

Reply:
Hi @ashimasingla20 his change is according to the documentation https://octokit.github.io/rest.js/#usageand if you see the issue #16 the warning message recommends do this

Reply:
Got it, Thanks.